### ISSUE TITLE
"skipping not structual" warning

### ISSUE THREAD

**Issue by MattMencel** [prometheus-community/json_exporter#6]
I've got a flat json like so:

```
{
  "account_type": "standard",
  "cash_credits": "22.20300",
  "city": "",
  "timezone": "America/Chicago"
}
```

My config.yml is this:

```
- name: plivo
  type: object
  path: $.cash_credits
  values:
    credits: $.cash_credits
```

when it gets parsed by the json_exporter, I get this warning and no metrics...

> WARN[2017-07-05T15:17:40-05:00] skipping not structual result;path:<&{$.cash_credits+ [%!s(*jsonpath.operator=&{3 0 0 false map[cash_credits:{}] [] [] []})] %!s(bool=true)}>,value:<"21.81800">

I'm guessing I've got something wrong in my config.yml, but not sure yet what I need to do.

**Comment by MattMencel** [comment]
OK...I think part of my problem was the type, so I switched the type to "value" and now get the "not numerical" error.

I also tried the fix in #5 and get the same "not numerical" warning.

**Comment by SuperQ** [comment]
This looks fixed.
